### PR TITLE
Walking skeleton

### PR DESCRIPTION
**Pull request by karen-olson**
Additions in this request: 

* A dedicated repository under version control that is accessible to the mentor team.
* A single passing test.
* A detailed README outlining how to run the app and tests.
* A continuous integration service is set up so that it runs the test suite upon every commit that is pushed.
* A simple message that is displayed on the console when the app is run.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+bundle exec rubocop -a
+```
+### Testing
+Run tests:
+```
+bundle exec rspec
+```
+To automate the test suite:
+```
+bundle exec guard init rspec
```
Comment (damonkelley) [inline README.md]: I am wondering if this is necessary? What does `init rspec` do? Is this something that users should run every time, or is this a one-time command needed to generate some boilerplate guard configuration?

**Review by damonkelley — COMMENTED** [review]
Looks good to me!

File: Gemfile [hunk 2]
```diff
@@ -4,7 +4,7 @@ source 'https://rubygems.org'
 
 # gem "rails"
 
-ruby "3.1.2"
+ruby '3.1.2'
```
Comment (kudukacar) [inline Gemfile]: Looks like the build failed because of a Ruby version mismatch.  Seems like it should be '3.1.0'.

**Comment by kudukacar** [comment]
Nice PR description and good incorporation of the linter.

File: README.md (hunk 1 above)
Comment (karen-olson) [reply README.md]: Good thought! I moved that command into the "installation" section.

File: Gemfile (hunk 2 above)
Comment (karen-olson) [reply Gemfile]: Thanks for pointing that out! I edited the version in the GitHub workflows ruby.yml file to match the project version.

Comment (karen-olson) [reply Gemfile]: I just saw that the ubuntu version specified in .github/workflows/ruby.yml doesn't support Ruby 3.1.2. I'm working on changing the project Ruby version to 3.1.0.

**Review by kudukacar — APPROVED** [review]
Looks great!

**Pull request merged**